v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,127 @@
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single [transaction](https://www.cockroachlabs.com/docs/dev/transactions.html). Executing statements in parallel helps reduce aggregate latency and improve performance. 
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+To understand why parallel statement execution is required, let's first understand sequential execution of SQL statements.
+
+Consider a scenario where we want to update a user's last name, favorite movie, and favorite song on a social networking application. Suppose the database has three tables: users, favorite_movies, and favorite_songs, that need to be updated to update the user information. Then the traditional transaction to update the user's information is as follows:
+
+~~~ sql
+> BEGIN;
+> UPDATE users SET last_name = 'Smith' WHERE id = 1;
+> UPDATE favorite_movies SET movies = 'The Matrix' WHERE user_id = 1;
+> UPDATE favorite_songs SET songs = 'All this time' WHERE user_id = 1;
+> COMMIT;
+~~~
+
+In this case, the server executes each statement to completion and sends the return value of each statement to the client. Only after the client receives the return value of a statement, it sends the next SQL statement to be executed. This is often described as a "conversational API," as demonstrated by the following conceptual diagram:
+
+<img src="{{ 'images/Sequential_Statement_Execution.png' | relative_url }}" alt="CockroachDB Parallel Statement Execution Error Mismatch" style="border:1px solid #eee;max-width:100%" />
+
+SQL engines traditionally execute the statements sequentially. In case of single-node databases, execution of the statements does not involve any communication latency. However, in case of globally-distributed CockroachDB clusters, statement execution results in communication latency. To maintain strong consistency across a globally-distributed cluster, CockroachDB replicates writes using the Raft [consensus protocol](https://www.cockroachlabs.com/blog/consensus-made-thrive/). The protocol requires a majority of replicas to agree before a write is committed. Replicas always live on separate nodes, and communication between these nodes takes time. Executing SQL statements sequentially, especially when each statement requires long-distance communication, leads to higher cumulative latency.
+
+With parallel statement execution, however, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency over multiple SQL statements. 
+
+## How Parallel Statement Execution Works
+
+SQL statements within a single transaction can be executed in parallel if the statements are independent. CockroachDB considers SQL statements within a single transaction to be independent if their execution can be safely reordered without affecting their results. 
```

Comment:
If we remove the scenario from "why" above, we'll need to lead into it in this section. Perhaps this is the section to the start with something like:

> To understand how parallel statement execution works, let's consider a scenario:
> - We want to update a user's last name, favorite movie, and favorite song on a social networking application.
> - The database has three tables that need to be updated: `users`, `favorite_movies`, and `favorite_songs`. 

And then intro the sequential/parallel diagrams and talk them through.